src/lib/ext-activitylog.js

```diff
@@ -226,8 +257,19 @@ class View {
     this.logView.clearTable();
   }
 
-  renderHeading({ tabId }) {
-    this.logHeading.textContent = `Activity Logs Filtered By Tab Id: ${tabId}`;
+  updatePageType(heading) {
+    this.pageType.textContent = heading;
+  }
+
+  updateMenus({ pageType }) {
+    if (pageType === 'load-logs') {
+      this.clearLogBtn.hidden = true;
+      this.optionsBtn.hidden = true;
+      this.menuWrapper
+        .querySelectorAll('.separator')
+        .forEach((eachSeparator) => (eachSeparator.hidden = true));
```

Comment:
I don't understand why the separator needs to be hidden unconditionally in this case. Why would the source of the log dictate whether separators are shown or not?